docs/content/SplitPageLayout.mdx

```diff
@@ -0,0 +1,313 @@
+---
+title: SplitPageLayout
+componentId: split_page_layout
+status: Draft
+description: Provides structure for a split page layout, including independent scrolling for the pane and content regions. Useful for responsive list/detail patterns, when an item in the pane updates the page content on selection.
+---
+
+<Note variant="warning">Not implemented yet</Note>
+
+## Examples
+
+### Default
+
+```jsx
+<SplitPageLayout>
+  <SplitPageLayout.Header>
+    <Placeholder label="Header" height={64} />
```

Comment:
What happens if I don't pass `height` to any of the placeholders? Is `Placeholder` a real component?

Reply:
Placeholder is a documentation-only component. The Header adapts the height of its content.